posthog/batch_exports/models.py

```diff
@@ -106,6 +106,13 @@ class Status(models.TextChoices):
     records_total_count = models.IntegerField(
         null=True, help_text="The total count of records that should be exported in this BatchExportRun."
     )
+    expected_records_count = models.IntegerField(
+        null=True,
+        help_text=(
+            "The total count of records that we expect to be exported in this BatchExportRun. "
+            "This is updated automatically post-batch export by a monitoring job."
+        ),
+    )
```

Comment:
We can just use `records_total_count` and save the migration. It's essentially the same field, and it's currently not in use. Feel free to update the `help_text`.

Reply:
ah nice, will do